lib/schemas/FieldSchema.js

```diff
@@ -132,7 +132,14 @@ module.exports = makeSchema(
         type: 'array',
         items: { $ref: '/FieldSchema' },
         description:
-          'An array of child fields that define the structure of a sub-object for this field. Usually used for line items.',
+          'An array of child fields that define the structure of a sub-array for this field. Used for line items.',
+        minItems: 1
+      },
+      fieldSet: {
+        type: 'array',
+        items: { $ref: '/FieldSchema' },
+        description:
+          'An array of fields that define the structure of a sub-object for this field. Used to group a set of fields into a logical section.',
```

Comment:
Do we want to re-enforce the idea this is a "visual" thing? Like "define the visual structure"?

Reply:
Do you mean in the description text? How about:
          'An array of fields that define the structure of a sub-object for this field. Used to group a set of fields into a logical section to display in the zap editor.',